### ISSUE TITLE
Black screen + UI

### ISSUE THREAD

**Issue by ragnarocko** [FTL13/FTL13#753]
![example](https://cloud.githubusercontent.com/assets/27042647/25567849/b057c21e-2dc4-11e7-8f43-ab1f391b6b9d.PNG)
**Problem Description**:
I joined the gamed the lobby screen loaded round was already going so I looked for a job found one joined but as soon as I joined I got a black screen but the UI was still there I cant right click to look at thing or move i think. But I can use UI elements move my pda turn on the light on it move the clip board and the like.

**What did you expect to happen**: 
Cause i couldnt move or see any game sprites.

**What happened instead**:
Basically its how i explained it.

**Why is this bad/What are the consequences:**
Keeps me from playing the round at all.

**Steps to reproduce the problem**:
I have no clue I joined in like normal the only thing I can think of thats diffrent was that i had AI as my highest job selection but that shouldnt matter.

**Possibly related stuff (which gamemode was it? What were you doing at the time? Was
anything else out of the ordinary happening?)**: I was told to update byond and Direct X and I did update byond but Direct X is alredy in the latest version possible.

**Comment by Cirrial** [comment]
Is this on a Windows machine?

**Comment by ragnarocko** [comment]
Yes yes it is.

**Comment by Mophise** [comment]
Why is there no word on a fix for this? We're having this problem on a private server. I've been looking through https://github.com/FTL13/FTL13/pull/797 and can't find anything on how to get my friends to the ship

**Comment by Vivalas** [comment]
Most probably because nobody knows the cause. BYOND issues like this are hard to track down and can be caused by a number of different reasons. And because they are hard to diagnose and fix, they also tend to fall down in the development priority.

I recommend asking on coderbus at /tg/ about this issue. There are some pretty crack developers over there, particularly @MrStonedOne, who might know more about why this happens. My guess is a resource issue, but I am guessing you probably have waited long enough to rule that out.

If you can do you mind right clicking next time this happens on the edge of the window and do client > options & messages? (or something like that), if there is a blue bar and "resources loading" or something similar, it means resources are still loading.

**Comment by Mophise** [comment]
have waited long enough to suffocate once.
have been teleported to the admin which instantly fixed it another time.
someone else died and wandered around as a ghost, and from the sounds of it, they were in an admin area they could only see after dying.
